gui/src/containers/BridgeSpec.js

```diff
@@ -0,0 +1,110 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import Typography from '@material-ui/core/Typography'
+import Grid from '@material-ui/core/Grid'
+import PaddedCard from 'components/PaddedCard'
+import Breadcrumb from 'components/Breadcrumb'
+import BreadcrumbItem from 'components/BreadcrumbItem'
+import Card from '@material-ui/core/Card'
+import { withStyles } from '@material-ui/core/styles'
+import { connect } from 'react-redux'
+import { bindActionCreators } from 'redux'
+import { fetchBridgeSpec } from 'actions'
+
+const styles = theme => ({
+  title: {
+    marginTop: theme.spacing.unit * 5,
+    marginBottom: theme.spacing.unit * 5
+  },
+  definitionTitle: {
+    marginBottom: theme.spacing.unit * 3
+  },
+  breadcrumb: {
+    marginTop: theme.spacing.unit * 5,
+    marginBottom: theme.spacing.unit * 5
+  }
+})
+
+const renderBridgeSpec = ({ classes, name, url, confirmations }) => (
+  <Grid container spacing={40}>
+    <Grid item xs={8}>
+      <PaddedCard>
+        <Typography variant='title' className={classes.definitionTitle}>
+          Bridge Info:
+        </Typography>
+        Name: {name} <br />
+        URL: {url} <br />
+        Confirmations: {confirmations} <br />
+      </PaddedCard>
+    </Grid>
+  </Grid>
+)
+
+const renderBridgeAssociations = ({ classes }) => (
```

Comment:
There is currently no API endpoint for associations so we should remove for this PR. We will add them in a separate story.